cli/cmd/plugin/unmanaged-cluster/tanzu/tanzu.go

```diff
@@ -597,6 +605,21 @@ func runClusterCreate(scConfig *config.UnmanagedClusterConfig) (*cluster.Kuberne
 	return kc, nil
 }
 
+func useExistingCluster(scConfig *config.UnmanagedClusterConfig) (*cluster.KubernetesCluster, error) {
+	scConfig.KubeconfigPath = scConfig.ExistingClusterKubeconfig
+	noopManager := cluster.NewNoopClusterManager()
+
+	kc, err := noopManager.Create(scConfig)
+	if err != nil {
+		return nil, err
+	}
+
+	log.Style(outputIndent, color.FgYellow).Warn("Warning: an existing cluster may not be compatible with kapp controller and the default CNI.")
+	log.Style(outputIndent, color.FgYellow).Warn("         unmanaged-cluster will not lifecycle this cluster. Use with caution.\n")
```

Comment:
**nit**: I like this warning, but would consider the following:

* remove the arbitrary indentation, or use a different value for `outputIndent`. TBH I don't think the indentation is needed at all with this one.
* remove "use with caution"
* I expect most users of this flag to put `--cni=none` so this warning won't be relevant for that case
* make the warning more about "components installed using this approach will need to be manually removed".